src/engine/server/sv_main.cpp

```diff
@@ -571,21 +573,25 @@ void SVC_Info( netadr_t from, const Cmd::Args& args )
 
 	SV_ResolveMasterServers();
 
-	// don't count privateclients
-	int botCount = 0;
-	int count = 0;
+	int bots = 0; // Bots always use public slots.
+	int publicSlotHumans = 0;
```

Comment:
maybe use players instead of humans?

Reply:
`Humans` does sound silly, but `Players` doesn't convey the idea of non-bot clients that I'm looking for.

Reply:
fair enough